tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -52,6 +52,17 @@ <h3 class="section-title">General</h3>
     </div>
   </div>
 
+  <div
+    class="control-row scalars-step-selector"
+    *ngIf="isScalarStepSelectorFeatureEnabled"
+  >
+    <mat-checkbox
+      [checked]="isScalarStepSelectorEnabled"
+      (change)="stepSelectorEnableToggled.emit()"
+      >Scalar Step Selector</mat-checkbox
```

Comment:
Still wondering should we make it more self-descriptive? The most important part is the data table so should we consider Nick's advice like
```
[ ] Enable step selection and data table 
    (Scalar only)
```